check-diff.sh

```diff
@@ -69,10 +70,16 @@ function set_environment_variables {
 			DIFF_BASE=${DIFF_BASE:-$TRAVIS_COMMIT^}
 		fi
 		DIFF_HEAD=${DIFF_HEAD:-$TRAVIS_COMMIT}
+	fi
+
+	if [[ -n "${BITBUCKET_BRANCH+set}" ]]; then
```

Comment:
The `+set` doesn't mean anything as far as I know. I think what you mean is:

```bash
[[ -n "${BITBUCKET_BRANCH}" ]]
```

Or equivalently:

```bash
[[ ! -z "${BITBUCKET_BRANCH}" ]]
```

As currently the `-n "${BITBUCKET_BRANCH+set}"` test will never be true, because `"${BITBUCKET_BRANCH+set}"` is interpolated as `"+set"` when the variable is empty, at least on Bash 3.2 on OSX.

Please make these changes and update the submodule in your PR to confirm they have the desired effect.